.github/workflows/build-platform-packages.yml

```
@@ -111,6 +111,13 @@
       contents: write
     
     steps:
+      - name: Debug GitHub context
+        run: |
```

actionlint (1.7.7)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)